brainscore_vision/model_helpers/activations/core.py

```diff
@@ -182,17 +305,25 @@ def _package(self, layer_activations, stimuli_paths):
             for coord in neuroid_coords:
                 neuroid_coords[coord][1] = np.concatenate((neuroid_coords[coord][1], layer_assembly[coord].values))
             assert layer_assemblies[0].dims == layer_assembly.dims
-            for dim in set(layer_assembly.dims) - {'neuroid'}:
-                for coord in layer_assembly[dim].coords:
-                    assert (layer_assembly[coord].values == nonneuroid_coords[coord][1]).all()
+            for coord, dims, values in walk_coords(layer_assembly):
+                if set(dims) == {'neuroid'}:
+                    continue
+                assert (values == nonneuroid_coords[coord][1]).all()
```

Comment:
these are equivalent right? Just a style rewrite?

Reply:
I believe so (I believe you made this change in person though)